### PR TITLE
meow meow meow meow

### PR DESCRIPTION
**Pull request by mcovarr**

### CODE REVIEW

File: src/main/scala/wdl4s/WdlNamespace.scala [hunk 1]
```diff
@@ -14,7 +15,6 @@ import wdl4s.values._
 import scala.collection.JavaConverters._
 import scala.language.postfixOps
 import scala.util.{Failure, Success, Try}
-import scalaz.Scalaz._
```
Comment (Horneth) [inline src/main/scala/wdl4s/WdlNamespace.scala]: 😄

**Comment by Horneth** [comment]
👍

**Comment by geoffjentry** [comment]
👍

**Comment by kshakir** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/wdl4s/pull-request/35/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/wdl4s)

**Pull request merged**